CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.11.0-rc0 (2024-04-19) - [#141]
```

Comment:
the link to a pr is not provided

Reply:
[link to pr](https://github.com/ayrat555/fang/pull/150/commits/d6c43c115f91b9b4598c48686f07015be54983a8)